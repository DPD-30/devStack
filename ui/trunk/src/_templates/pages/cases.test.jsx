import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesCases from './cases.jsx'

jest.unmock('./cases.jsx')


describe('', () => {
    it('PagesCases should match snapshot', () => {
        const comp = renderToString(<PagesCases />)

        expect(comp).toMatchSnapshot()
    })
})
