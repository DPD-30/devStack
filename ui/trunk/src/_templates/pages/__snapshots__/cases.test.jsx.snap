// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesCases should match snapshot 1`] = `
<div data-reactroot>
  <div class="justify-content-md-center row">
    <h1>
      <strong>
        Cases
      </strong>
    </h1>
    <button type="button"
            style="margin:10px;border-radius:10px;height:min-content"
            class="btn btn-primary btn-sm"
    >
      Add
    </button>
  </div>
</div>
`;
